### ISSUE TITLE
New GUI to display and allow editing of word list?

### ISSUE THREAD

**Issue by Theclaws** [ManiacDC/TypingAid#45]
Now that the SQL version "hides" the wordlist in a database, do we need a GUI front end for view and editing database contents?  

As the functionality of frequency/weights gets added to the learned words database, it may not be practical to blow away the database and reload from a text file... so a GUI front end may be helpful here.

**Comment by ManiacDC** [comment]
Yes, I had it in mind to create one. Thanks for creating an issue for this.

**Comment by Theclaws** [comment]
For those who might be interested and comfortable working with databases, you could use SQLite Studio to view and edit the database of words.   But use with extreme caution... And a backup before editing... In sure ManiacDC won't support you if you accidentally mess the database up. :-)  http://sqlitestudio.pl
